addons/payment_ogone/controllers/main.py

```diff
@@ -50,7 +50,7 @@ def feedback(self, **kwargs):
         payment = request.registry.get('payment.transaction')
         try:
             tx = payment._ogone_form_get_tx_from_data(cr, uid, kwargs, context=context)
-            payment._ogone_s2s_validate(tx)
+            payment._ogone_s2s_validate_tree(tx, kwargs)
```

Comment:
Will break when payment_ogone/models/ogone.py:423 is executed since it is waiting a lxml.etree.ElementTree object and not a dict.
```python
'html_3ds': str(tree.HTML_ANSWER).decode('base64')
```